DevoxxClientMobile/src/main/java/com/devoxx/util/DevoxxNotifications.java

```diff
@@ -187,7 +187,7 @@ public void preloadingScheduledSessionsDone() {
         }
         
         // add the notification for new ones if they haven't started yet
-        if (dateTimeStart != null && dateTimeStart.isAfter(now)) {
+        if (dateTimeStart.isAfter(now)) {
```

Comment:
Few lines above it is created as
`ZonedDateTime dateTimeVote = session.getEndDate().plusMinutes(SHOW_VOTE_NOTIFICATION);`, so it either throws NPE on `getEndDate` or it is never null.